src/main/java/de/retest/recheck/ui/actions/ActionState.java

```diff
@@ -36,8 +36,9 @@ private ActionState() {
 	}
 
 	public ActionState( final Action action, final SutState resultingState, final long duration ) {
-		assert action != null;
-		assert resultingState != null;
+		if ( action == null || resultingState == null ) {
+			throw new IllegalArgumentException( "action and resultingState should not be null." );
```

Comment:
Please throw a correct NPE. Or use lombok `@NonNull` with a lombok constructor. 

This class should not be used anymore!?